api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -339,6 +339,8 @@ public static String getOpenmrsProperty(String property) {
 	
 	public static final String GLOBAL_PROPERTY_PATIENT_SEARCH_MATCH_START = "START";
 	
+	public static final String GLOBAL_PROPERTY_PATIENT_SEARCH_MATCH_SOUNDEX = "Soundex";
```

Comment:
Shouldn't this be "SOUNDEX" to match the other types here?

Reply:
Done